pkg/router/functionServiceMap.go

```diff
@@ -79,5 +83,7 @@ func (fmap *functionServiceMap) assign(f *metav1.ObjectMeta, serviceURL *url.URL
 
 func (fmap *functionServiceMap) remove(f *metav1.ObjectMeta) error {
 	mk := keyFromMetadata(f)
+
+	fmap.logger.Info("removing key and value from functionServiceMap")
```

Comment:
Do you know how often these logs will be emitted?

I'm worried we're going to blow our Axiom ingest if it's very high 😅

Reply:
No clue, I guess i'll remove the logs mutating the map